modelica_builder/builder.py

```diff
@@ -126,3 +126,91 @@ def build(self):
             annotations = f" annotation({', '.join(self._annotations)})"
 
         return f'\n\tconnect({self._a}, {self._b}){annotations};\n\t'
+
+
+class ParameterBuilder:
+    def __init__(self, insert_index, type_, identifier):
+        """ParameterBUilder allows you to construct and insert a parameter
+
+        :param insert_index: int, index to place the new parameter. if < 0, it will insert at the end
+        :param type_: string, type of the parameter
+        :param identifier: string, parameter identifier
+        """
+        self._arguments = {}
+        self._value = None
+        self._annotations = []
+        self._string_comment = None
+        self._insert_index = insert_index
+        self._type = type_
+        self._identifier = identifier
+
+    def set_argument(self, arg_name, arg_value):
+        """set_argument sets a parameter initialization argument
+
+        :param arg_name: string, name of the argument
+        :param arg_value: string, value of the argument
+        """
+        self._arguments[arg_name] = arg_value
+
+    def set_value(self, value):
+        """set_value sets the value of the parameter, e.g. Real myParam=10.0
+
+        :param value: string, value to set
+        """
+        self._value = value
```

Comment:
looks like this is really only setting the 10.0? right?

Reply:
That's correct, I can clarify in the comments